From 0a87c9b9d64ed6df42c64f86fd8dedb7911fcf0e Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Wed, 11 Jul 2012 15:33:49 -0400
Subject: [PATCH 1/2] autogen.sh: Add /bin/sh interpreter, honor NOCONFIGURE=1

First, we should be able to execute as "./autogen.sh".  Second,
add support for NOCONFIGURE=1.  For more information,
see http://people.gnome.org/~walters/docs/build-api.txt
---
 autogen.sh | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/autogen.sh b/autogen.sh
index 69e765a..4ebafa9 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -9,5 +9,7 @@ if [ $# -gt 0 ] ; then
 		shift 1
 		;;
 	esac
-    exec ./configure  "$@"
+	if test -z "$NOCONFIGURE"; then
+		exec ./configure  "$@"
+	fi
 fi
-- 
1.8.3.1

